state/redis/redis.go

```diff
@@ -125,15 +161,26 @@ func (r *StateStore) Init(metadata state.Metadata) error {
 		}
 	}
 
-	r.client = redis.NewClient(opts)
-	_, err = r.client.Ping().Result()
-	if err != nil {
-		return fmt.Errorf("redis store: error connecting to redis at %s: %s", m.host, err)
+	return redis.NewClient(opts)
+}
+
+func (r *StateStore) newFailoverClient(m metadata) *redis.Client {
+	opts := &redis.FailoverOptions{
+		MasterName:      r.metadata.sentinelMasterName,
+		SentinelAddrs:   []string{r.metadata.host},
+		DB:              defaultDB,
+		MaxRetries:      m.maxRetries,
+		MaxRetryBackoff: m.maxRetryBackoff,
 	}
 
-	r.replicas, err = r.getConnectedSlaves()
+	/* #nosec */
```

Comment:
Please elaborate on this code comment?

Reply:
That's a command to tell the linter to skip a check here